src/lib/content/stats/author.js

```diff
@@ -9,10 +9,10 @@ function getAuthor()
   let author = null;
 
   // schema.org
-  let element = document.querySelector("[itemprop='author']");
+  let element = document.querySelector("[property='author' i],[itemprop='author' i]");
```

Comment:
Coding style: Please add a space before the second selector. Unfortunately, the linter doesn't catch that automatically.